ca/ca_test.go

```diff
@@ -229,8 +229,10 @@ func setup(t *testing.T) *testCtx {
 
 	crl, err := NewCRLImpl(
 		boulderIssuers,
-		boulderProfile.Lints,
-		time.Hour,
+		issuance.CRLProfileConfig{
+			ValidityInterval: config.Duration{Duration: 216 * time.Hour},
```

Comment:
I think it was @beautifulentropy who provided a nice distinction:

 - period: a span of time, unanchored from any particular instance (i.e. a duration)
 - interval: a span of time anchored to a specific beginning and end time
 
 So `ValidityPeriod` would be nice here.